controller/network/terminator.go

```diff
@@ -93,49 +112,36 @@ func (ctrl *TerminatorController) Create(s *Terminator) (string, error) {
 	return id, err
 }
 
-func (ctrl *TerminatorController) createInTx(ctx boltz.MutateContext, e *Terminator) (string, error) {
-	if e.Id == "" {
-		var err error
-		e.Id, err = ctrl.sequence.NextHash()
-		if err != nil {
-			return "", err
-		}
-	}
-	if e.Binding == "" {
-		if strings.HasPrefix(e.Address, "udp:") {
-			e.Binding = "udp"
+func (ctrl *TerminatorController) checkBinding(terminator *Terminator) {
+	if terminator.Binding == "" {
+		if strings.HasPrefix(terminator.Address, "udp:") {
+			terminator.Binding = "udp"
 		} else {
-			e.Binding = "transport"
+			terminator.Binding = "transport"
```

Comment:
Is this going to break Xgress for non-`udp` and non-`transport` bindings?

Reply:
It shouldn't. It only takes effect if a binding is not specified. The edge is now aware of bindings, so a binding will only use this logic if the user doesn't provide a binding on purpose (as opposed to previously when they couldn't provide a binding).

Longer term, now that the edge can provide this, we could consider taking it out and letting the tooling do this.